aptos-move/framework/move-stdlib/sources/configs/features.move

```diff
@@ -387,6 +419,15 @@ module std::features {
         byte_index < vector::length(features) && (*vector::borrow(features, byte_index) & bit_mask) != 0
     }
 
+    fun apply_diff(features: &mut Features, enable: vector<u64>, disable: vector<u64>) {
+        vector::for_each_ref(&enable, |feature| {
```

Comment:
Use vector::for_each instead of for_each_ref so you wouldn't need to do *feature